zaas-service/src/main/java/org/zowe/apiml/zaas/zaas/ZaasExceptionHandler.java

```diff
@@ -52,8 +72,8 @@ public ResponseEntity<ApiMessageView> handleSafIdtExceptions(RuntimeException ex
             .body(messageView);
     }
 
-    @ExceptionHandler(value = {ApplicationNameNotFoundException.class})
-    public ResponseEntity<ApiMessageView> handleApplIdNotFoundException(ApplicationNameNotFoundException ex) {
+    @ExceptionHandler(value = {ApplicationNameNotFoundException.class, HttpMessageNotReadableException.class})
```

Comment:
Why is `HttpMessageNotReadableException` also showing the invalid applid message?

Reply:
in case you do not provide a message body, it will produce this exception